### DISCUSSION TITLE
Compatibility between 1.5.5 release and 0.74.2 fw

### DISCUSSION THREAD

**Discussion by ilyabreev** [akopachov/flipper-zero_authenticator#60] [Q&A] [answered]
Hello!

Tried to run version 1.5.5 of an application on 0.74.2 official firmware and got "Api version mismatch" error. Are they compatible?

**Comment by akopachov** [comment, marked as answer]
Hey, 
give a try to version [1.5.6](https://github.com/akopachov/flipper-zero_authenticator/releases/tag/v1.5.6) it built based of latest official firmware. Also make sure you are using right FAP file, there are two prebuilt files published:
* [totp_official-stable_fw.fap](https://github.com/akopachov/flipper-zero_authenticator/releases/download/v1.5.6/totp_official-stable_fw.fap) - for stable firmware
* [totp_official-dev_unleashed_fw.fap](https://github.com/akopachov/flipper-zero_authenticator/releases/download/v1.5.6/totp_official-dev_unleashed_fw.fap) for dev firmware (also compatible with Unleashed and other non-official firmwares).

**Reply by ilyabreev** [reply]
Thanks, worked like a charm!
